js/hello-mixit.controller.js

```diff
@@ -0,0 +1,15 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('hello-mixit', [])
+        .controller('HelloMixitCtrl', HelloMixitCtrl);
+
+    function HelloMixitCtrl() {
+        var vm = this;
+
+        vm.greetings = "Hello World !";
+
+    }
+
```

Comment:
Can we just use `this` and remove `vm` variable?
Can we also fix those unnecessary new lines?